stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/CaptureMediaContract.kt

```diff
@@ -34,11 +34,13 @@ public class CaptureMediaContract : ActivityResultContract<Unit, File>() {
                 videoFile = it
                 createIntentList(context, MediaStore.ACTION_VIDEO_CAPTURE, it)
             }
-        return Intent.createChooser(Intent(), context.getString(R.string.stream_input_camera_title))
+        val intents = takePictureIntents + recordVideoIntents
+        val initialIntent = intents.lastOrNull() ?: Intent()
+        return Intent.createChooser(Intent(initialIntent), context.getString(R.string.stream_input_camera_title))
```

Comment:
Why do we need to wrap `initialIntent` with `Intent()`?

Reply:
Ummm, it is already an Intent, you are right.
Let me check if I can remove it and then update the PR